.github/workflows/build.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Build
-        run: clang -Werror -fsanitize=fuzzer libfuzzer-dotnet-windows.cc -o libfuzzer-dotnet-windows.exe
+        run: clang -Werror -O2 -fsanitize=fuzzer libfuzzer-dotnet-windows.cc -o libfuzzer-dotnet-windows.exe
```

Comment:
What's the difference between O2 and the default level?

Reply:
Better code optimizations. I saw that Onefuzz used it without any problems with Libfuzzer-dotnet.